patch add to resolve the following build issue post the glib2 update

Undefined                       first referenced
 symbol                             in file
g_object_unref                      paps-paps.o  (symbol belongs to implicit dependency /usr/lib/amd64/libgobject-2.0.so.0)
g_option_group_add_entries          paps-paps.o  (symbol belongs to implicit dependency /usr/lib/amd64/libglib-2.0.so.0)
g_utf8_get_char                     paps-paps.o  (symbol belongs to implicit dependency /usr/lib/amd64/libglib-2.0.so.0)

--- a/src/Makefile.am	2019-12-26 17:00:25.230217747 -0800
+++ b/src/Makefile.am	2019-12-26 17:00:36.652695995 -0800
@@ -3,7 +3,7 @@
 bin_PROGRAMS = paps
 paps_CFLAGS  = $(WARN_CFLAGS) $(PANGO_CFLAGS)
 paps_SOURCES = paps.c
-paps_LDADD =  $(WARN_LDFLAGS) $(PANGO_LIBS) $(lib_LIBRARIES) $(all_libraries)
+paps_LDADD =  $(WARN_LDFLAGS) $(PANGO_LIBS) $(lib_LIBRARIES) $(all_libraries) -lglib-2.0 -lgobject-2.0
 paps_DEPENDENCIES = $(lib_LIBRARIES)
 
 AM_CPPFLAGS = -DGETTEXT_PACKAGE='"$(GETTEXT_PACKAGE)"' -DDATADIR='"$(datadir)"'
